dataloader.properties

```diff
@@ -101,6 +101,7 @@ loginUrl=https://rest.bullhornstaffing.com/rest-services/login
 #stateTaxFormExistField=customText1
 #taskExistField=taskUUID
 #tearsheetExistField=name
+#workersCompensationExistField=name
```

Comment:
You'll also want the same thing for:

```
#workersCompensationRateExistField=______
```

But we really need a string field that we can use to identify it, so that we can use the data loader to update it. If we don't then we will need to skipUpdates in the integration test.

Reply:
There is no string field for WorkersCompRate so I did update the integration tests.